ngraph/frontend/onnx_import/src/op/instance_norm.cpp

```diff
@@ -70,7 +70,7 @@ namespace ngraph
                         bias.get_element_type(),
                         ").");
 
-                    if (data_pshape.rank().is_static())
+                    if (data_pshape.is_static())
```

Comment:
Maybe you can leave this check for dynamic shapes too, but with `compatible` instead of `same_shame` as you've done it for broadcast.

Reply:
Nice idea, implemented